analyzer/src/funTest/assets/projects/synthetic/gomod/go.sum

```diff
@@ -0,0 +1,20 @@
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
It's not needed, as it's not a lockfile - it's a security feature. Would we be more fault tolerant (to changes in the hashes) if we didn't commit that file?

Reply:
Like I said in the commit message, it's added for convenience. The alternatives would be to add it to gitignore or to delete it in a finally block in the test, but compared to that I prefer committing the file. Also, the hashes should never change.